docs/authentication/jwt-auth.html

```diff
@@ -16,13 +16,407 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="../../css/reset.css">
     <link rel="stylesheet" href="../../css/docs.css">
+    <link rel="stylesheet" media="screen and (max-width: 900px)" href="../../css/docs_mobile.css">
     <link href=“https://fonts.googleapis.com/css?family=IBM+Plex+Sans” rel=“stylesheet”>
   </head>
   <body>
-    <h1 class="heading-1">JWT Authentication</h1>
+      <main>
+      <h1 class="heading-1">JSON Web Token Authentication</h1>
+      <p class="block-text">A JSON Web Token (JWT) defines a compact and self-contained way for securely transmitting information between parties as a JSON object. You can find out more about JWTs at `JWT.io`.</p>
+      <p class="block-text">This guide uses JWT with Swift-JWT to implement Single Sign On (SSO) authentication for your Kitura routes.</p>
+      <h2 class="heading-2">Prerequisites</h2>
+      <ul class="plain-list">
+        <li><a onclick="setActiveSidebarElementForParent('create-server')" href="../getting-started/create-server.html#">Kitura Server:</a> Learn how to create one in our Getting Started guide</li>
+        <li><a onclick="setActiveSidebarElementForParent('style-guide')" href="../getting-started/style-guide.html#">Kitura Style Guide:</a> Follow our guide to see how a production ready server should be structed</li>
+        <li><a onclick="localStorage.setItem('package', 'Swift-JWT'); setActiveSidebarElementForParent('swift-jwt')" href="../getting-started/update-package.html#">Update Dependencies:</a> Swift-JWT needs adding to the Package.swift file.</li>
+      </ul>
+      <div class="underline"></div>
+      <h2 class="heading-2">JWT Authentication</h2>
+      <h2 class="heading-2"><span class="blue-text">Step 1:</span> Create a JWTRoutes.swift file</h2>
+      <p>Open your `Sources` > `Application` > `Application.swift`</p>
+      <p> Inside the `postInit()` function add: </p>
+      <pre><code class="language-swift">initializeJWTRoutes(app: self)</code></pre>
+      <p>Create a new file called `JWTRoutes.swift` in `Sources` > `Application` > `Routes`</p>
+      <p>Inside this file, add the framework for our routes code:</p>
+      <pre><code class="language-swift">import Foundation
+import KituraContracts
```

Comment:
Can we assume users have `KituraContracts` as a dependency in their `Package.swift` file or does it need to be listed in our dependencies? It isn't clear why we need it at this point either.

Reply:
KituraContracts is brought in By Kitura so they will have it. It is used for step 9 for request error. Since it will be requires by most guide I didn't feel like we need to repeat the reason for it in each one.